### PR TITLE
Updating boolean flag doc examples

### PR DESCRIPTION
**Pull request by willvedd**
### 🔧 Changes

Example usage of boolean flags had very subtle errors where it would advise `--flag true` instead of `--flag=true` or simply `--flag`. Similarly with setting the values of these flags as false, would advise `--flag false` instead of `--flag=false`. This PR corrects these errors and where appropriate, provides examples of how to set certain boolean flags to false.

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/cli/rules.go [hunk 1]
```diff
@@ -307,10 +307,10 @@ func updateRuleCmd(cli *cli) *cobra.Command {
 			"To update interactively, use `auth0 rules update` with no arguments.\n\n" +
 			"To update non-interactively, supply the rule id and other information through the flags.",
 		Example: `  auth0 rules update <id>
-  auth0 rules update <rule-id> --enabled true
-  auth0 rules update <rule-id> --enabled true --name "My Updated Rule"
-  auth0 rules update <rule-id> --enabled true --name "My Updated Rule" --script "$(cat path/to/script.js)"
-  auth0 rules update <rule-id> -e true -n "My Updated Rule" -s "$(cat path/to/script.js)" --json
+  auth0 rules update <rule-id> --enabled=true
```
Comment (Widcket) [inline internal/cli/rules.go:310]: Maybe let's not use examples that have `=true` (the default when the flag is passed without an explicit bool value), and only use `=` for `=false` examples?
Like in the examples for `auth0 users import` in the file below.

Comment (Widcket) [reply internal/cli/rules.go:310]: Same for all the other instances.

Comment (willvedd) [reply internal/cli/rules.go:310]: Negating the  `--enabled` flag is ambiguous, could be `--disabled` could be `--enabled false` or could be `--enabled=false`. Only one of which is correct. Why not make the template explicitly clear in the example to eliminate any confusion? Can you explain the upside you see by not doing that?

Comment (Widcket) [reply internal/cli/rules.go:310]: Using bool flags like the following:
- Case "true": `--enabled`
- Case "false": `--enabled=false`
Follows the existing conventions for bool flags in other CLIs, as far as I can tell. But of course, feel free to not do it if you disagree.

Comment (Widcket) [reply internal/cli/rules.go:310]: Just to be clear, what I'm suggesting here is to use this form: `--enabled` for "true" examples, and this other form: `--enabled=false` for "false" examples, like in the docs for `auth0 users import`.

**Review by Widcket — APPROVED** [review]
Not a blocker.

**Pull request merged**